src/app/shared/services/skin/skin.service.ts

```diff
@@ -57,13 +66,57 @@ export class SkinService {
         this.appConfigService.APP_CONFIG.APP_FIELDS.APP_SKIN,
         targetSkin.name
       );
+      this.updateRoutingDefaults(targetSkin, oldSkin);
     } else {
       console.error(`No skin found with name "${skinName}"`, {
         availableSkins: this.availableSkins,
       });
     }
   }
 
+  /** Override changes to config-dependent routing config inherited in app-routing.module */
+  private updateRoutingDefaults(newSkin?: IAppSkin, oldSkin?: IAppSkin) {
+    const newRouteDefaults = newSkin?.appConfig?.APP_ROUTE_DEFAULTS;
+    let routes = this.router.config;
+    if (newRouteDefaults) {
```

Comment:
Checks like this require that the skin's `appConfig` consists of a minimal set of overrides. If the skin's `appConfig` contains the whole default `appConfig`, then this check will always pass (assuming that the default appConfig includes `APP_ROUTE_DEFAULTS`). 
See other comment on the debug skin file